descartes_planner/test/sparse_planner.cpp

```diff
@@ -37,7 +40,7 @@ class ThreeDOFRobot: public descartes_trajectory_test::CartesianRobot
 {
 public:
   ThreeDOFRobot():
-    descartes_trajectory_test::CartesianRobot(0,0,3)
+    descartes_trajectory_test::CartesianRobot(0,0)
```

Comment:
Almost all of the code (minus the type names) is duplicated between these files.